kv/txn_coord_sender_test.go

```diff
@@ -443,30 +455,31 @@ func TestTxnCoordSenderCleanupOnAborted(t *testing.T) {
 func TestTxnCoordSenderGC(t *testing.T) {
 	s := createTestDB(t)
 	defer s.Stop()
-	kv := s.DB.InternalKV()
-	coord := getCoord(kv)
 
 	// Set heartbeat interval to 1ms for testing.
-	coord.heartbeatInterval = 1 * time.Millisecond
+	s.Sender.heartbeatInterval = 1 * time.Millisecond
 
 	txn := newTxn(s.Clock, proto.Key("a"))
-	if err := kv.Run(client.Call{
+	call := client.Call{
 		Args:  createPutRequest(proto.Key("a"), []byte("value"), txn),
-		Reply: &proto.PutResponse{}}); err != nil {
+		Reply: &proto.PutResponse{},
+	}
+	s.Sender.Send(context.TODO(), call)
+	if err := call.Reply.Header().GoError(); err != nil {
```

Comment:
I see this pattern repeat itself quite often in this file. How about creating a method that calls out like:

if err := SendPut(createPutRequest(proto.Key("a", []byte("value"), txn)); err != nil {